roles/openshift_node/tasks/config.yml

```diff
@@ -79,12 +79,24 @@
   set_fact:
     l_release_image: "{{ oc_get.stdout }}"
 
-- name: Pull release image
-  command: "podman pull --tls-verify={{ openshift_node_tls_verify }} --authfile {{ temp_dir.path }}/pull-secret.json {{ l_release_image }}"
+- name: Check for cluster proxy
+  k8s_facts:
```

Comment:
We removed all the usages of k8s_* modules so we didn't have the requirement for python2-openshift on the control host.  I'd prefer to not reintroduce k8s_* modules unless there is a specific need.

Ref: https://github.com/openshift/openshift-ansible/pull/11452

Reply:
Doh. Maybe I was working on a (very) outdated branch but I thought I saw other instances. NP, will remove.

Reply:
We still have k8s stuff in the test/ directory that I have not gotten back to.  That may be where you saw it.  Since that code is only run in CI, we can easily make sure dependencies are installed without having the customer install something else.  Also, I've seen minor issues with the module and it was just something else to troubleshoot.  Going for 'simple' as the lowest common denominator.